docs/bql.md

```diff
@@ -407,7 +407,7 @@ that would require multiple clauses and extra bindings:
 ```
 
 Note that the intervals defined by `before`, `after`, and `between` are always closed (the 
-limits of the intervals are included). You can then understand the `after` as a '>=', the 
+limits of the intervals are included). You can then understand the `after` as a `>=`, the 
```

Comment:
should this be in another commit/PR?

Reply:
I agree, thank you Kloss. I have just extracted this minor correction to a separate commit.